demo_unicycle_coordination_circle_jjc.py

```diff
@@ -0,0 +1,139 @@
+import sys
```

Comment:
Same as before. Maybe we should update the demo in the main repository instead of adding new ones.